pkg/syncer/k8scloudoperator/k8scloudoperator.go

```diff
@@ -229,6 +229,8 @@ func (k8sCloudOperator *k8sCloudOperator) getPod(ctx context.Context, pvcName st
 			if pvClaim != nil && pvClaim.ClaimName == pvcName {
 				log.Debugf("Returned pod: %s with pvClaim name: %s and namespace: %s running on node: %s",
 					spew.Sdump(&pod), pvcName, pvcNamespace, nodeName)
+				log.Infof("Returned pod: %s/%s with pvClaim name: %s running on node: %s",
```

Comment:
No. In some cases, the pod spec need to be printed out. But that would be very rare case. So we can keep it in DEBUG level. But in most cases, the info log added in Line 232 should be sufficient to debug. We need add this because in prod env, the log level is not set to DEBUG.

Reply:
In that case, can you just dump Pod spec in the debug log, since the Info level log will also be printed when log level is set to development? using the TraceId we can identify the log.

Reply:
Fixed.